src/lib/slider/slider.ts

```diff
@@ -518,7 +518,7 @@ export class MatSlider extends _MatSliderMixinBase
   _onSlideEnd() {
     this._isSliding = false;
 
-    if (this._valueOnSlideStart != this.value) {
+    if (this._valueOnSlideStart != null && this._valueOnSlideStart != this.value) {
```

Comment:
I don't think this is quite right, how about: `this._valueOnSlideStart != this.value && !this.disabled`

Reply:
Yes, maybe is better, I am not in context at all. I will make the change (Y)

Reply:
I have made the change @mmalerba